packages/presentation/lib/src/star_movie_app.dart

```diff
@@ -17,21 +19,29 @@ class _StarMovieAppState extends BlocScreenState<StatefulWidget, AppBloc> {
     return MaterialApp(
       title: 'Star Movie',
       theme: ThemeData.from(colorScheme: dark),
-      home: StreamBuilder(
+      home: StreamBuilder<BaseTile<AppData>>(
         stream: bloc.dataStream,
         builder: (context, snapshot) {
-          final baseTile = snapshot.data as BaseTile?;
+          final baseTile = snapshot.data;
           final tile = baseTile?.tile;
-          if (tile == null)
+          if (tile == null || baseTile == null)
             return Center(
               child: CircularProgressIndicator(),
             );
-          return Navigator(
-            onPopPage: (route, result) {
-              bloc.handleRemoveRouteSettings(route.settings);
-              return route.didPop(result);
-            },
-            pages: tile.pages.toList(),
+          return Scaffold(
+            bottomNavigationBar: baseTile.isLoading
+                ? null
+                : AppNavigationBar(
+                    bloc: bloc,
+                    bottomNavIndex: tile.bottomNavIndex,
+                  ),
```

Comment:
up to u, but from ux side it seems strange when while loading u hide bottom navigation